### ISSUE TITLE
Make scroll_preview and "go in windows" customizable

### ISSUE THREAD

**Issue by Curs3W4ll** [nvimdev/lspsaga.nvim#1236]
**Is your feature request related to a problem? Please describe.**
I would like to have different key binds than the default ones to go up/down in code actions list and focus code actions window.

**Describe the solution you'd like**
An option to allow custom keys for these actions?

**Describe alternatives you've considered**
None

**Comment by glepnir** [comment]
`scroll_preview` support scroll in preview or peek window. like finder.

**Comment by Curs3W4ll** [comment]
But how can I change these bindings, adding this in the config works?
```
{
    diagnostics = {
        scroll_preview = {
            prev = "<C-n>",
            next = "<C-N>",
        },
    },
}
```

**Comment by glepnir** [comment]
```lua
finder.scroll_preview = {  prev , next }
```

**Comment by Curs3W4ll** [comment]
Thanks, that was what I wanted to know, is this somewhere in the doc? I can't find it.

If it is not, it would be cool to add it 😄 

Thanks for the fast replies, best regards.

**Comment by Curs3W4ll** [comment]
I tried using the following config:

```lua
code_action = {
    scroll_preview = {"<C-c>N", "<C-c>n" },
},
```

But this don't seems to work, did I did something wrong? :/

**Comment by glepnir** [comment]
this will take effect all preview window which support preview.

> ```lua
> finder.scroll_preview = {  prev , next }
> ```

**Comment by Curs3W4ll** [comment]
Defining `finder.scroll_preview` seems to do nothing for me.

However I search a bit by myself, and while running `:lua print(vim.inspect(require("lspsaga")))`, I noticed this: `config.scroll_preview = { scroll_down = "<C-f>", scroll_up = "<C-b>" }`.

So I used the following config: 
```lua
require("lspsaga").setup({
    scroll_preview = {
        scroll_down = "<C-c>n",
        scroll_up = "<C-c>N",
})
```

And this works like a charm!

I don't know if it was intended to by used like that, and if I was something wrong with the solution you were proposing to me, but this works.

However I still have one question, can I change the keybind used to focus the diagnostic window (which is `<C-w>w` by default), is this a Neovim keybind rather that a lspsaga one?
